src/transports/ws/sws.c

```diff
@@ -759,7 +835,7 @@ static int nn_sws_fail_conn (struct nn_sws *self, int code, char *reason)
         nn_fsm_raise (&self->fsm, &self->done, NN_SWS_RETURN_CLOSE_HANDSHAKE);
     }
 
-    return 0;
+    return;
```

Comment:
IIRC, it's something that pedantic compilers prefer; either way, it doesn't hurt to be explicit. Do you have strong opionions otherwise? Else, we'll keep it for now.